src/main/scala/md/fusionworks/adam/jbrowse/models/convert/Convert.scala

```diff
@@ -0,0 +1,61 @@
+package md.fusionworks.adam.jbrowse.models.convert
+import org.apache.spark.{SparkConf, SparkContext}
+import org.bdgenomics.adam.rdd.{ADAMContext, ADAMRDDFunctions}
+
+class Convert {
```

Comment:
Change class to object. Change name to smth like "AdamConvertionUtil" or "AdamConvertor"